src/conformance/conformance_test/test_LayerComposition.cpp

```diff
@@ -349,21 +350,25 @@ namespace Conformance
                 const float t = y / 255.0f;
                 const XrColor4f dst = Colors::Green;
                 const XrColor4f src{0, 0, t, t};
-                const XrColor4f blended{dst.r * (1 - src.a) + src.r, dst.g * (1 - src.a) + src.g, dst.b * (1 - src.a) + src.b, 1};
+                const XrColor4f blended{dst.r * (1 - src.a) + src.r, dst.g * (1 - src.a) + src.g, dst.b * (1 - src.a) + src.b, 0};
```

Comment:
nit: could use a comment to explain the 0 alpha is to catch runtimes which assume alpha blending

Reply:
Added such a comment.